### PR TITLE
TransactionTests Update: ttEIP2028

### PR DESCRIPTION
**Pull request by marioevz**
Contains updated TransactionTests for category ttEIP2028.

### CODE REVIEW

File: src/TransactionTestsFiller/ttEIP2028/DataTestSufficientGas2028Filler.json [hunk 1]
```diff
@@ -1,38 +1,29 @@
...
+    {
+        "expectException" : 
+        {
+            "Frontier" : "TR_IntrinsicGas",
+            "Homestead" : "TR_IntrinsicGas",
+            "EIP150" : "TR_IntrinsicGas",
+            "EIP158" : "TR_IntrinsicGas",
+            "Byzantium" : "TR_IntrinsicGas",
+            "Constantinople" : "TR_IntrinsicGas",
+            "ConstantinopleFix" : "TR_IntrinsicGas"
```
Comment (winsvega) [inline src/TransactionTestsFiller/ttEIP2028/DataTestSufficientGas2028Filler.json]: `>= Istanbul` not explicit that we don't mark validity here, but ok. just need to remember that tests are against all forks

File: src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json [hunk 2]
```diff
@@ -1,34 +1,31 @@
...
+        "expectException" : 
+        {
+            "Frontier" : "TR_IntrinsicGas",
+            "Homestead" : "TR_IntrinsicGas",
+            "EIP150" : "TR_IntrinsicGas",
+            "EIP158" : "TR_IntrinsicGas",
+            "Byzantium" : "TR_IntrinsicGas",
+            "Constantinople" : "TR_IntrinsicGas",
+            "ConstantinopleFix" : "TR_IntrinsicGas",
+            ">=Istanbul" : "TR_IntrinsicGas"
```
Comment (winsvega) [inline src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json]: then you can mark `>= Frontier`

**Review by winsvega — COMMENTED** [review]
minor comments

File: src/TransactionTestsFiller/ttEIP2028/DataTestSufficientGas2028Filler.json (hunk 1 above)
Comment (marioevz) [reply src/TransactionTestsFiller/ttEIP2028/DataTestSufficientGas2028Filler.json]: Updated this in the latest commit.

File: src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json (hunk 2 above)
Comment (marioevz) [reply src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json]: Updated this in the latest commit.

**Pull request merged**